pom.xml

```diff
@@ -362,7 +363,7 @@
       <dependency>
         <groupId>org.jctools</groupId>
         <artifactId>jctools-core</artifactId>
-        <version>1.2</version>
+        <version>1.2.1</version>
```

Comment:
With this update I believe we can now update [MAX_ALLOWED_MPSC_CAPACITY](https://github.com/netty/netty/blob/4.1/common/src/main/java/io/netty/util/internal/PlatformDependent.java#L95) to be `1 << 30`? @normanmaurer - please confirm.